docs/docs/command-line-interface.mdx

```diff
@@ -52,83 +52,79 @@ This creates the following files:
 
-This command trains a Rasa model that combines a Rasa NLU and a Rasa Core model.
-If you only want to train an NLU or a Core model, you can run `rasa train nlu` or `rasa train core`.
-However, Rasa will automatically skip training Core or NLU if the training data and config haven't changed.
+If you want to train an NLU or dialogue model individually, you can run
+`rasa train nlu` or `rasa train core`. If you provide training data only for one one of
+these, `rasa train` will fall back to one of these commands by default.
 
-`rasa train` will store the trained model in the directory defined by `--out`. The name of the model
-is per default `<timestamp>.tar.gz`. If you want to name your model differently, you can specify the name
-using `--fixed-model-name`.
+`rasa train` will store the trained model in the directory defined by `--out`, `models/` by default.
+The name of the model by default is `<timestamp>.tar.gz`. If you want to name your model differently,
+you can specify the name using the `--fixed-model-name` flag.
 
 The following arguments can be used to configure the training process:
 
 ```text [rasa train --help]
 ```
 
-:::note
-Make sure training data for Core and NLU are present when training a model using `rasa train`.
-If training data for only one model type is present, the command automatically falls back to
-`rasa train nlu` or `rasa train core` depending on the provided training files.
-
-:::
-
 ## rasa interactive
 
-To start an interactive learning session with your assistant, run
+To start an interactive learning session with your assistant, run:
 
 ```bash
 rasa interactive
 ```
 
-If you provide a trained model using the `--model` argument, the interactive learning process
-is started with the provided model. If no model is specified, `rasa interactive` will
-train a new Rasa model with the data located in `data/` if no other directory was passed to the
-`--data` flag. After training the initial model, the interactive learning session starts.
-Training will be skipped if the training data and config haven't changed.
+This will first train a model and then start an interactive shell session.
+You can then correct your assistants predictions as you talk to it.
 
-During the interactive learning, Rasa will plot the current conversation
+If you provide a trained model using the `--model` argument, training is skipped
+and that model will be loaded instead.
+
+During interactive learning, Rasa will plot the current conversation
 and a few similar conversations from the training data to help you
-keep track of where you are. You can view the visualization at
-[http://localhost:5005/visualization.html](http://localhost:5005/visualization.html)
-as soon as you've started interactive learning.
-This plotting can take some time; to skip the visualization, run `rasa interactive --skip-visualization`.
+keep track of where you are. You can view the visualization
+[in your browser](http://localhost:5005/visualization.html)
```

Comment:
not sure if this makes sense as people likely won't have it running when they click the link